tox.ini

```diff
@@ -35,5 +35,4 @@ deps=mypy
 setenv=MYPYPATH={toxinidir}
 # TODO: Drop --ignore-missing-imports once we have type annotations for eth_utils, coincurve and cytoolz
 commands=
-    mypy --py2 --ignore-missing-imports -p eth_keys
     mypy --ignore-missing-imports -p eth_keys
```

Comment:
While you're at it, would it make sense to add `--check-untyped-defs --disallow-incomplete-defs` here as well?